app/views/users/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= current_user.email%>
+
+<h1>Events I Created</h1>
+<p>Click on an event title to see the details </p>
+<ul>
+  <% current_user.events.each do |event| %>
+    <li>
+       <%= link_to event.even_name, event_path(event) %>
+    </li> 
+  <% end %>
+</ul>
+
+<h1>Events Am Attending </h1>
+<p>Click on an event title to see the details </p>
+<ul>
+  <% current_user.attended_events.each do |event| %>
```

Comment:
**REQUIRED**
There should be no logic in the views. 
We can have `attended_events` as a class instance variable in UsersController#index action and access it directly from the views.
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright